src/components/Root/index.module.scss

```diff
@@ -44,4 +44,10 @@ body {
 body > div:first-child {
   display: flex;
   min-height: 100%;
-}
\ No newline at end of file
+}
+
+html[class="darkMode"],
+html[class="darkMode"] div[style*="background-image"],
+html[class="darkMode"] img:not([id="safe-logo"]) {
+  filter: invert(1) hue-rotate(180deg);
```

Comment:
I don't necessarily agree with this because it is not interfacing with the theme. It is more 'high contrast mode'.

@liliiaorlenko wanted to put together a dark palette and then we actually use the [`dark` theme in MUI](https://v4.mui.com/customization/color/#color).

Reply:
so should I already provide the palette?

Reply:
Creating a dark theme that uses MUI will need to be done incrementally. We will need the palette though to do so.

I don't know what the final decision should be on this ticket as it merely inverts colours. It won't need the palette here though.

Reply:
@liliiaorlenko no, no rush, it was just a quick experiment.

@iamacook it's good enough for me personally, easier on the eyes, but I agree it's not a good long-term solution.

Reply:
@iamacook renamed to hi-contrast:
<img width="356" alt="Screenshot 2022-01-10 at 17 41 55" src="https://user-images.githubusercontent.com/381895/148803753-73719d4e-94b5-4679-b574-0e003d9a8211.png">